{"uid":"22e149aed6e4571b","name":"test_expose_external","fullName":"tests.integration.test_expose_external#test_expose_external","historyId":"7750323f37c67d2243b6ad910c6b49b1","time":{"start":1745113364349,"stop":1745113367345,"duration":2996},"description":"Test the expose-external config option.","descriptionHtml":"<p>Test the expose-external config option.</p>\n","status":"broken","statusMessage":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_expose_external.py\", line 113, in test_expose_external\n    await asyncio.gather(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1982, in deploy\n    return await self._deploy(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 2359, in _deploy\n    raise JujuError(\"\\n\".join(errors))\njuju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1745113364149,"stop":1745113364150,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1745113364150,"stop":1745113364150,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1745113364150,"stop":1745113364349,"duration":199},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"charm","time":{"start":1745113364349,"stop":1745113364349,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1745113364349,"stop":1745113364349,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test the expose-external config option.","status":"broken","statusMessage":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","statusTrace":"Traceback (most recent call last):\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n                                ^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n            ^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n           ^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.12/asyncio/base_events.py\", line 687, in run_until_complete\n    return future.result()\n           ^^^^^^^^^^^^^^^\n  File \"/root/spread_project/tests/integration/test_expose_external.py\", line 113, in test_expose_external\n    await asyncio.gather(\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 1982, in deploy\n    return await self._deploy(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/root/spread_project/.tox/integration/lib/python3.12/site-packages/juju/model.py\", line 2359, in _deploy\n    raise JujuError(\"\\n\".join(errors))\njuju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","steps":[],"attachments":[{"uid":"25aabb2f4c27b5da","name":"log","source":"25aabb2f4c27b5da.txt","type":"text/plain","size":1021}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1745113367353,"stop":1745113367353,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1745113367360,"stop":1745113367955,"duration":595},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1745113367956,"stop":1745113367956,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1745113367957,"stop":1745113367957,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1745113367957,"stop":1745113367957,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_expose_external"},{"name":"host","value":"fv-az1979-73"},{"name":"thread","value":"19793-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_expose_external"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"8f5f4867c3a42013","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[]}],"history":{"statistic":{"failed":0,"broken":10,"skipped":0,"passed":117,"unknown":3,"total":130},"items":[{"uid":"628668650719b9bc","reportUrl":"../2761//#testresult/628668650719b9bc","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1745027028918,"stop":1745027031365,"duration":2447}},{"uid":"3e9a97bac640e0f5","reportUrl":"../2760//#testresult/3e9a97bac640e0f5","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1744940596407,"stop":1744940598397,"duration":1990}},{"uid":"119d0047d5946409","reportUrl":"../2759//#testresult/119d0047d5946409","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1744854147924,"stop":1744854150328,"duration":2404}},{"uid":"ded47e5b873fefcb","reportUrl":"../2752//#testresult/ded47e5b873fefcb","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1744767927866,"stop":1744767931008,"duration":3142}},{"uid":"67ae694766093353","reportUrl":"../2736//#testresult/67ae694766093353","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1744681560918,"stop":1744681564615,"duration":3697}},{"uid":"f5ad770eaaa55feb","reportUrl":"../2733//#testresult/f5ad770eaaa55feb","status":"unknown","time":{}},{"uid":"871f5521056c98b2","reportUrl":"../2732//#testresult/871f5521056c98b2","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1744421916996,"stop":1744421919646,"duration":2650}},{"uid":"19de26b30d987430","reportUrl":"../2711//#testresult/19de26b30d987430","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1744335765998,"stop":1744335768768,"duration":2770}},{"uid":"270152f9f804dab3","reportUrl":"../2703//#testresult/270152f9f804dab3","status":"broken","statusDetails":"juju.errors.JujuError: base \"ubuntu@22.04/stable\" is not supported, supported bases are: ubuntu@24.04","time":{"start":1744249434172,"stop":1744249436357,"duration":2185}},{"uid":"23968a90cfc2692f","reportUrl":"../2702//#testresult/23968a90cfc2692f","status":"passed","time":{"start":1744163041595,"stop":1744163188902,"duration":147307}},{"uid":"f62403007ced7865","reportUrl":"../2701//#testresult/f62403007ced7865","status":"passed","time":{"start":1744076909065,"stop":1744077057739,"duration":148674}},{"uid":"4af40d4f12ced6b4","reportUrl":"../2698//#testresult/4af40d4f12ced6b4","status":"passed","time":{"start":1743990155012,"stop":1743990297918,"duration":142906}},{"uid":"55feb437cf5d11d","reportUrl":"../2697//#testresult/55feb437cf5d11d","status":"passed","time":{"start":1743903781364,"stop":1743903928616,"duration":147252}},{"uid":"c830296994e617dd","reportUrl":"../2696//#testresult/c830296994e617dd","status":"passed","time":{"start":1743817354231,"stop":1743817497544,"duration":143313}},{"uid":"1c8064cd25a5b527","reportUrl":"../2695//#testresult/1c8064cd25a5b527","status":"broken","statusDetails":"juju.errors.JujuUnitError: Unit in error: mysql-router-k8s/0","time":{"start":1743731093395,"stop":1743731161380,"duration":67985}},{"uid":"2cbc54cee9622611","reportUrl":"../2686//#testresult/2cbc54cee9622611","status":"passed","time":{"start":1743644580669,"stop":1743644721452,"duration":140783}},{"uid":"8eb403c4d1f8311f","reportUrl":"../2680//#testresult/8eb403c4d1f8311f","status":"passed","time":{"start":1743558412843,"stop":1743558558374,"duration":145531}},{"uid":"febdab7a1874f7e8","reportUrl":"../2664//#testresult/febdab7a1874f7e8","status":"passed","time":{"start":1743472179874,"stop":1743472324197,"duration":144323}},{"uid":"803f558b9ae1c8ae","reportUrl":"../2661//#testresult/803f558b9ae1c8ae","status":"passed","time":{"start":1743385427889,"stop":1743385573537,"duration":145648}},{"uid":"d710646a2755c47b","reportUrl":"../2660//#testresult/d710646a2755c47b","status":"passed","time":{"start":1743298950669,"stop":1743299097935,"duration":147266}}]},"tags":["asyncio","abort_on_fail"]},"source":"22e149aed6e4571b.json","parameterValues":[]}